keyboards/kbdfans/kbd8x_mk2/keymaps/default/readme.md

```diff
@@ -0,0 +1,4 @@
+# The default keymap for KBD8X MKII
+
+A typical setup. Nothing special.
```

Comment:
```
# The default keymap for KBD8X MKII

A typical setup. Nothing special.
A nice starting point for customizing.
```

In here you have these lines as two lines 3 and 4 however will render onto one lines if this is what you want thats fine however if not double space is required
```suggestion
A typical setup. Nothing special.  
```